src/plugins/vue-i18n.ts

```diff
@@ -4,7 +4,7 @@ export default () => {
     silentFallbackWarn: true,
     pluralizationRules: {
       /**
-       * @param choice - a choice index given by the input to $tc: `$tc('path.to.rule', choiceIndex)`
+       * @param choice - a choice index given by the input to $tc: $tc('header.filter-button.with-count', choiceIndex)
```

Comment:
`vue-i18n-extract` was reporting `path.to.rule` key as missing even though it's in the comments. So, I replaced it with a key that actually exists.